packages/js/product-editor/src/products-app/router.tsx

```diff
@@ -56,9 +57,16 @@ export default function useLayoutAreas() {
 				content: <ProductList />,
 				preview: false,
 				mobile: <ProductList postType={ postType } />,
-				edit: showQuickEdit && (
-					<ProductEdit postType={ postType } postId={ postId } />
-				),
+				edit:
+					showQuickEdit &&
+					( isListLayout ? (
+						<ProductEditWithOldForm
+							postType={ postType }
+							postId={ postId }
+						/>
+					) : (
+						<ProductEdit postType={ postType } postId={ postId } />
+					) ),
```

Comment:
Is there a particular reason for this logic?

Reply:
No, only based off of some of the initial design explorations where we show the full form with the list view, and we actually also do something similar in GB where we show the preview editor in the list view only: https://github.com/WordPress/gutenberg/blob/141e9cd884053db9baec26e43297162841f9e7a7/packages/edit-site/src/components/site-editor-routes/pages.js#L42
<img width="805" alt="Screenshot 2024-12-09 at 3 58 51 PM" src="https://github.com/user-attachments/assets/62e81678-8de9-4a21-bac3-c6c8466eb00e">